### DISCUSSION TITLE
Discord: Chat with brackets community

### DISCUSSION THREAD

**Discussion by abose** [brackets-cont/brackets#75] [Announcements]
Chat with the Brackets community on discord for any support using this link. https://discord.com/invite/rBpTBPttca

[![image](https://user-images.githubusercontent.com/5336369/142358928-a506c466-f19d-44f6-aab1-1c137ee68278.png)](https://discord.com/invite/rBpTBPttca)

**Comment by kevgray** [comment]
How do you use the "." notation to add classes to elements. Is there a tutorial somewhere ???

**Reply by chrismcintosh78** [reply]
hw do i install brackts from source?

**Comment by HAMZA6669** [comment]
appdata/roaming/brackets.io/brackets2.0.1/install/brackets1.cab

**Comment by HAMZA6669** [comment]
appdata/roaming/brackets.io/brackets2.0.1/install/brackets1.cab


secure file not found
